src/main/resources/config/run_test_popgen.xml

```diff
@@ -1,13 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
+<!-- This file is used by the popgen tests; edit at your own risk. -->
+
 <run>
     <rng_seed>1</rng_seed>
     <r0>11</r0>
     <seeding_rate>0.002</seeding_rate>
     <immunity_rate>0.8</immunity_rate>
-    <geodistribution_profile>belgium_population.csv</geodistribution_profile>
-    <population_file>population_model_default.xml</population_file>
+    <geodistribution_profile>belgium_population_major.csv</geodistribution_profile>
+    <population_file>population_model_test.xml</population_file>
     <reference_households>households_flanders.json</reference_households>
-    <num_days>50</num_days>
+    <num_days>10</num_days>
```

Comment:
Haha~ I figured a *population-generation-testing* config file shouldn’t spend so long running a simulation! Verifying that the simulation runs at all is fine.